src/react-components/preferences-screen.js

```diff
@@ -484,6 +494,18 @@ const preferenceLabels = defineMessages({
     id: "preferences-screen.preference.show-rtc-debug-panel",
     defaultMessage: "Show RTC Panel"
   },
+  showAudioDebugPanel: {
+    id: "preferences-screen.preference.show-audio-debug-panel",
+    defaultMessage: "Show Audio Debug Panel"
+  },
+  enableAudioClipping: {
+    id: "preferences-screen.preference.enable-audio-clipping",
+    defaultMessage: "Enable Audio Clipping"
```

Comment:
So we removed them from the preferences panel and we don't apply clipping at all unless you are in the audio debugging mode (until we have spoke/blender integration)? Or we default it as disabled?

Reply:
My argument is even once we decide to use clipping this would not be a user preference but one related to the scene audio properties and falloff settings for audio zones and such. Not sure this makes sense as a user pref.